src/ray/common/cgroup2/tests/BUILD.bazel

```diff
@@ -31,3 +32,22 @@ ray_cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+# This test is run through sysfs_cgroup_driver_integration_test_entrypoint.sh
+ray_cc_test(
+    name = "sysfs_cgroup_driver_integration_test",
+    srcs = ["sysfs_cgroup_driver_integration_test.cc"],
+    tags = [
+        "cgroup",
+        "no_windows",
```

Comment:
I think you can use the bazel flag I sent you on slack to have this only run on linux instead of blacklisting windows